corehq/apps/userreports/views.py

```diff
@@ -385,7 +385,7 @@ def import_report(request, domain):
 def report_source_json(request, domain, report_id):
     config = get_document_or_404(ReportConfiguration, domain, report_id)
     config._doc.pop('_rev', None)
-    return json_response(config)
+    return json_response(config._doc)
```

Comment:
This doesn't seem like the right fix to me. Something inside config._doc is not actually json serializable somehow, and I think we need to figure out what that is. Anything that's json serializable can be `deepcopy`d and it's essentially failing on `deepcopy(config._doc)`.